code/modules/power/lighting.dm

```diff
@@ -200,34 +236,32 @@
 
 /obj/machinery/light/built/New()
 	status = LIGHT_EMPTY
-	update(0)
```

Comment:
Why did you remove the 0 here and from the others below?

Reply:
The `0` is implied if there's no arguments, so I thought it'd be cleaner to just remove them.

Reply:
well in the proc: `/obj/machinery/light/proc/update(trigger = TRUE)`, the default arg is true which means now all of the old `update(0)`'s in this file are `update(TRUE)`.

Reply:
Just tested it again and yeah, you're right.

The last time I tested it it said that `trigger` was set to `null`, but I guess I was looking at the wrong thing?